### PR TITLE
TDP-3395 Implement GET /calls/{callId} endpoint

### PR DESCRIPTION
**Pull request by muesono-daitan**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `gitbook` 

- [ ] I made sure that the site looks great and functions perfectly live
- [ ] I ran splell check
- [ ] I'm ready for these changes to be made live

### General Review

- [ ] All changes/updates requested were intentionally changed or added (no extraneous file or partial updates)
- [ ] The live rendered page behaves and looks like intended.
- [ ] All links work as intended. (click every link to make sure that it takes the user to the expected place).
- [ ] The markdown was rendered to HTML as intended. (Tables look right, code samples are gated properly).
- [ ] Any style or CSS changes are tested on multiple different pages to ensure nothing unexpected broke or changed.
- [ ] I proof-read the live site (`gitbook serve`) and there are no oblivious splelling or grammer misteaks.

### Gitbook Specific

- [ ] Any new documentation pages have been added to the `SUMMARY.md` file so they are rendered to HTML.
- [ ] Any links to other pages within this documentation use relative links (`[read more about ...](guides/voice/voicemail.md)`).
- [ ] Gitbook can install and build the documentation (`gitbook install` & `gitbook build`)

### Code Specific

- [ ] All code has been tested against the live API.
- [ ] All code can be run without throwing errors or **new** warnings.
- [ ] All code is formatted correctly and consistently (spaces, tabs).
- [ ] All code is legible and idiomatic to the language.
- [ ] All code uses sensical method, function, and variable names.
- [ ] All code samples use the appropriate syntax highlighting.
- [ ] Any examples re-use same phone numbers, uuids, variable names, etc... throughout (IE don't have something like `{from: '+19191231234', to: '+19191231234'}` unless it's meant to show the same phone number calling/texting itself ).

## TL;DR

- [ ] I did these things
- [ ] I'm ready for these changes to be made live

### CODE REVIEW

File: voice/methods/calls/about.md [hunk 1]
```diff
@@ -16,10 +16,5 @@ Read More about Calls and Voice in the <a href="http://dev.bandwidth.com/faq/#vo
...
-| <code class="get">GET</code>       | [`/calls/{callId}/recordings`](getCallsCallIdRecordings.md)                                     | Retrieve information about all of the recordings that occurred during a call |
-| <code class="get">GET</code>       | [`/calls/{callId}/recordings/{recordingId}`](getCallsCallIdRecordingsRecordingId.md)            | Retrieve information about a recording                                       |
-| <code class="delete">DELETE</code> | [`/calls/{callId}/recordings/{recordingId}`](deleteCallsCallIdRecordingsRecordingId.md)         | Delete a recording                                                           |
-| <code class="get">GET</code>       | [`/calls/{callId}/recordings/{recordingId}/media`](getCallsCallIdRecordingsRecordingIdMedia.md) | Retrieve a recording                                                         |
-| <code class="get">GET</code>       | [`/recordings`](getRecordings.md)                                                               | Returns a max of 1000 recordings, sorted by startTime from oldest to newest  |
```
Comment (muesono-daitan) [inline voice/methods/calls/about.md:25]: The pages pointed by those links were moved to another path and referenced from another page. These links here are pointing to the wrong path and seems they were forgotten.

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3540.s3-website-us-east-1.amazonaws.com/

File: voice/methods/recordings/getRecordings.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ curl -X GET \
     "transcriptionUrl"    : null
   },
   {
-    "accountId"           : "5552319",
+    "accountId"           : "5555555",
```
Comment (muesono-daitan) [inline voice/methods/recordings/getRecordings.md]: These are actual `accountId`s.

File: voice/methods/calls/getCallsCallId.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  "from"                : "+15551112222",
+  "direction"           : "inbound",
+  "state"               : "disconnected",
+  "startTime"           : "2019-10-21T16:45:11.293Z",
+  "answerTime"          : "2019-10-21T16:45:14.352Z",
+  "endTime"             : "2019-10-21T16:55:12.950Z",
+  "disconnectCause"     : "hangup",
+  "errorMessage"        : null,
+  "errorId"             : null,
+  "lastUpdateTime"      : "2019-10-21T16:55:12.950Z"
```
Comment (muesono-daitan) [inline voice/methods/calls/getCallsCallId.md]: This `lastUpdateTime` field was added but we are open to suggestions. Does this name look alright to everyone?

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3540.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3540.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3540.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3540.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3540.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3540.s3-website-us-east-1.amazonaws.com/

File: SUMMARY.md [hunk 4]
```diff
@@ -90,15 +90,16 @@
...
 		* [PUT Pause/Resume Recording](voice/methods/recordings/putCallsCallIdRecording.md)
 	* [Recording Management](voice/methods/recordings/about.md)
-		* [PUT Pause/Resume Recording](voice/methods/recordings/putCallsCallIdRecording.md)
 		* [GET Account Recordings](voice/methods/recordings/getRecordings.md)
+		* [PUT Pause/Resume Recording](voice/methods/recordings/putCallsCallIdRecording.md)
 		* [GET Call Recordings](voice/methods/recordings/getCallsCallIdRecordings.md)
 		* [GET Recording Information](voice/methods/recordings/getCallsCallIdRecordingsRecordingId.md)
-		* [GET Download recording](voice/methods/recordings/getCallsCallIdRecordingsRecordingIdMedia.md)
 		* [DELETE Delete Recording](voice/methods/recordings/deleteCallsCallIdRecordingsRecordingId.md)
+		* [GET Download recording](voice/methods/recordings/getCallsCallIdRecordingsRecordingIdMedia.md)
```
Comment (muesono-daitan) [inline SUMMARY.md:107]: Changed the order to match the order in _voice/methods/recordings/about.md_

File: voice/methods/calls/getCallsCallId.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
+{% method %}
+
+## Retrieve Call Information
+Retrieve the current state of a specific call.
+
+The `state` attribute indicates the current state of the call. Valid values are `created`, `answered` and `disconnected`.
```
Comment (muesono-daitan) [inline voice/methods/calls/getCallsCallId.md]: The states here are different from the states in Catapult (`started` / `rejected` / `active` / `completed` / `transferring`). I chose those based on the names of the callback events we send: `Answer` / `Disconnect`, but these state names can be changed if anyone opposes to them.

Comment (muesono-daitan) [reply voice/methods/calls/getCallsCallId.md]: I've been thinking about changing the `created` state to `initiated` since we got an `Initiate` callback.

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3540.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3540.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3540.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3540.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3540.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3540.s3-website-us-east-1.amazonaws.com/

**Review by jmulford-bw — APPROVED** [review]
Approving from a DX point of view assuming that the content is accurate 👍

File: voice/methods/calls/getCallsCallId.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+## Retrieve Call Information
+Retrieve the current state of a specific call.
+
+The `state` attribute indicates the current state of the call. Valid values are `initiated`, `answered` and `disconnected`.
+
+The `answerTime` attribute is populated once the call has been answered.
+
+The `endTime` and `disconnectCause` attributes are populated once the call has ended.
+
+The `errorMessage` and `errorId` attributes are populated only if the call ends with an error.
```
Comment (n-miles) [inline voice/methods/calls/getCallsCallId.md]: Can we add a table containing all fields and their descriptions?

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3540.s3-website-us-east-1.amazonaws.com/

**Comment by muesono-daitan** [comment]
This will be merged after we get feedback from the customer.

**Review by jchavez443 — APPROVED** [review]

**Pull request merged**